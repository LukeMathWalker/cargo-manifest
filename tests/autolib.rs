use cargo_manifest::Manifest;

#[test]
fn autolib() {
    let m = Manifest::from_path("tests/autolib/Cargo.toml").expect("load autolib");
    insta::assert_debug_snapshot!(m);
}
