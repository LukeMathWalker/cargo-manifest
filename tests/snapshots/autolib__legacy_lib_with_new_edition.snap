---
source: tests/autolib.rs
expression: error
---
can't find library, rename file to `src/lib.rs` or specify lib.path
